doc/helpers/Input-extensions.md

```diff
@@ -8,11 +8,12 @@ Provides various attached properties for _input controls_, such as `TextBox` and
 
 ## Attached Properties
 
-| Property               | Type      | Description                                                                                |
-|------------------------|-----------|--------------------------------------------------------------------------------------------|
-| `AutoDismiss`          | `bool`    | Whether the soft-keyboard will be dismissed when the enter key is pressed.                 |
-| `AutoFocusNext`        | `bool`    | Whether the focus will move to the next focusable element when the enter key is pressed.\* |
-| `AutoFocusNextElement` | `Control` | Sets the next control to focus when the enter key is pressed.\*                            |
+| Property               | Type         | Description                                                                                                           |
+|------------------------|--------------|-----------------------------------------------------------------------------------------------------------------------|
+| `AutoDismiss`          | `bool`       | Whether the soft-keyboard will be dismissed when the enter key is pressed.                                            |
+| `AutoFocusNext`        | `bool`       | Whether the focus will move to the next focusable element when the enter key is pressed.\*                            |
+| `AutoFocusNextElement` | `Control`    | Sets the next control to focus when the enter key is pressed.\*                                                       |
+| `ReturnType`           | `ReturnType` | The return button type on the soft-keyboard. Can be one of the following: __Default, Done, Go, Next, Search, Send__.  |
```

Comment:
```suggestion
| `ReturnType`           | `ReturnType` | The type of return button on a soft keyboard. It can be one of the following options: __Default, Done, Go, Next, Search, Send__.  |
```